content/docs/glossary/clawback.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Clawbacks
+order:
+---
+
+import { CodeExample } from "components/CodeExample";
+import { Alert } from "components/Alert";
+
+Protocol 17 introduces operations that allow asset issuers to maintain tighter control over how their asset is distributed to the world. Specifically, it gives them power to **burn their asset** from a trustline or claimable balance, effectively removing it from the recipient's balance sheet:
+
+> The amount of the asset clawed back is burned and is not sent to any other address. The issuer may reissue the asset to the same account or to another account if the intent of the clawback is to move the asset to another account.
+
+This allows for things like regulatory enforcement, safety and control over certain assets, etc. You can refer to [CAP-35](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0035.md) for more motivations or technical details behind these new features.
+
+
+## Relevant operations
+Configuring and performing a clawback involves several discrete steps:
+
```

Comment:
Possible to clarify who takes each of these steps?  I believe 
- An issuer sets a clawback flag on their issuing account
- An asset holder establishes a trustline for the asset
- Any account can make a payment with the asset
- The issuer can claw back some or all of the asset

But I think the flow will be easier to follow if the actor is more explicit.

Reply:
Good call! Done in e52ee4a1c4da4f314eb728e0a337c8554718a74e.